mysql-test/include/search_pattern_in_file.inc

```diff
@@ -0,0 +1,98 @@
+# Purpose:
```

Comment:
This file was ported from mariadb (and refactored by robert). The testcase using it is also from mariadb.
The difference compared to search_pattern.inc is that versions treats the file as a text file, reading it line by line (this version treats it as binary), and this in_file version also aborts the test immediately - the search_file in 8.0 only prints out a message, and the test fails at the end because of a diff in the output.

Reply:
Can we add a binary or text option to search_pattern.inc, and then re-record for the abort-vs-result difference?